scripts/deploy-mainnet.sh

```diff
@@ -1,32 +1,32 @@
 #!/bin/sh
 
-UniswapV2CalleeDai=$(dapp create UniswapV2CalleeDai \
+echo "UniswapV2CalleeDai:"
+forge create UniswapV2Callee.sol:UniswapV2CalleeDai --constructor-args \
   0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D \
-  0x9759A6Ac90977b93B58547b4A71c78317f391A28)
+  0x9759A6Ac90977b93B58547b4A71c78317f391A28
 
-UniswapV2LpTokenCalleeDai=$(dapp create UniswapV2LpTokenCalleeDai \
+echo "UniswapV2LpTokenCalleeDai:"
+forge create UniswapV2LpTokenCallee.sol:UniswapV2LpTokenCalleeDai --constructor-args \
   0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D \
-  0x9759A6Ac90977b93B58547b4A71c78317f391A28)
+  0x9759A6Ac90977b93B58547b4A71c78317f391A28
 
-UniswapV3Callee=$(dapp create UniswapV3Callee \
+echo "UniswapV3Callee:"
+forge create UniswapV3Callee.sol:UniswapV3Callee --constructor-args \
   0xE592427A0AEce92De3Edee1F18E0157C05861564 \
-  0x9759A6Ac90977b93B58547b4A71c78317f391A28)
+  0x9759A6Ac90977b93B58547b4A71c78317f391A28
 
-WstETHCurveUniv3Callee=$(dapp create WstETHCurveUniv3Callee \
+echo "WstETHCurveUniv3Callee.sol:WstETHCurveUniv3Callee:"
+forge create WstETHCurveUniv3Callee.sol:WstETHCurveUniv3Callee --constructor-args \
   0xDC24316b9AE028F1497c275EB9192a3Ea0f67022 \
   0xE592427A0AEce92De3Edee1F18E0157C05861564 \
   0x9759A6Ac90977b93B58547b4A71c78317f391A28 \
-  0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)
+  0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
 
-CurveLpTokenUniv3Callee=$(dapp create CurveLpTokenUniv3Callee \
+echo "CurveLpTokenUniv3Callee:"
+forge create CurveLpTokenUniv3Callee.sol:CurveLpTokenUniv3Callee --constructor-args \
   0xE592427A0AEce92De3Edee1F18E0157C05861564 \
   0x9759A6Ac90977b93B58547b4A71c78317f391A28 \
-  0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2)
+  0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2
```

Comment:
I feel like the output is less clean like this, but I used to publish these back when we had a keeper coordination channel.  Since that doesn't exist anymore, I'll just register my protest, but not hold this PR up because of it.

Reply:
Looks like we just need to change the `mainnet` ones to: `src/<file>.sol:<file>Dai`.  Where the mainnet ones are missing `src/`.